src/NUnitFramework/tests/Attributes/TheoryTests.cs

```diff
@@ -42,15 +42,15 @@ public void TheoryWithNoArgumentsIsTreatedAsTest()
         }
 
         [Test]
-        public void TheoryWithNoDatapointsIsNotRunnable()
+        public void TheoryWithNoDatapointsIsRunnable()
         {
-            TestAssert.IsNotRunnable(fixtureType, "TheoryWithArgumentsButNoDatapoints");
+            TestAssert.IsRunnable(fixtureType, "TheoryWithArgumentsButNoDatapoints");
         }
 
         [Test]
-        public void UnsupportedNullableTypeArgumentWithNoDatapointsAreNotRunnable()
+        public void UnsupportedNullableTypeArgumentWithNoDatapointsAreRunnable()
```

Comment:
Nit: should be `UnsupportedNullableTypeArgumentWithNoDatapointsIsRunnable` because it's really talking about running the test, not running the datapoints which is the object of the preposition 'with' which is modifying 'runnable.'